runtime/queries/json/highlights.scm

```diff
@@ -19,10 +19,9 @@
  "{" "}"
 ] @punctuation.bracket
 
-(("\"" @conceal)
+(("\"" @string)
  (#set! conceal ""))
 
-(escape_sequence) @string.escape
-((escape_sequence) @conceal
- (#eq? @conceal "\\\"")
+((escape_sequence) @string.escape
+ (#eq? @string.escape "\\\"")
```

Comment:
These aren't equivalent queries. We still need to separate them
```suggestion
(escape_sequence) @string.escape
((escape_sequence) @string.escape
 (#eq? @string.escape "\\\"")
```

Reply:
This could also use `@_conceal`
```clojure
((escape_sequence) @_conceal
 (#eq? @_conceal "\\\"")
 ```